master/internal/trial.go

```diff
@@ -153,6 +154,21 @@ func (t *trial) Receive(ctx *actor.Context) error {
 				return fmt.Errorf("persisting trial in prestart: %w", err)
 			}
 		}
+
+		if t.continued {
+			// TODO these should be in a transaction eventually.
+			err := t.addTask()
+			if err != nil {
+				return err
+			}
+			if _, err := db.Bun().
+				NewInsert().
+				Model(&model.TrialTaskID{TrialID: t.id, TaskID: t.taskID}).
```

Comment:
question (and this existed before this pr) why isn't this already part of addTask? do we have a case where we add a task to the trial but don't want to add the corresponding row here?

Reply:
I think this is mostly here for historical reasons since we used to persist the trial much later than we persisted the task.